testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/asserts/AttributesAssert.groovy

```diff
@@ -41,7 +41,7 @@ class AttributesAssert {
     } else if (value instanceof Closure) {
       assert ((Closure) value).call(val)
     } else {
-      assert val == value
+      assert value == val
```

Comment:
Why?

Reply:
No particular reason. I tried to get nice assertion messages out of here, which would indicate for which attribute the assertion failed, but I failed eventually as various places across the project were failing with all the solutions I tried. So eventually I returned it in the initial implementation, but sides got switched incidentally. Will put it back as it was.